use_case_examples/resnet/README.md

```diff
@@ -104,14 +104,14 @@ GPU machine: 8xH100 GPU machine
 
 Summary of the accuracy evaluation on ImageNet (100 images):
 
-| w&a bits | p_error | Accuracy | Top-5 Accuracy | Runtime*        | Device |
```

Comment:
If someone could explain, why I had to do a `make mdformat`. I would I appreciate it. Because it's breaking my PR
If someone could explain to me why I needed  `make mdformat`. I'd appreciate it. Because it breaks my PR